src/Basis/Constructors.jl

```diff
@@ -36,8 +36,11 @@ function sausage_transformation(d)
     c = zeros(d + 1)
     c[2:2:end] = [1, cumprod(1:2:d-2) ./ cumprod(2:2:d-1)...] ./ (1:2:d)
     c /= sum(c)
-    g = Polynomial(c)
-    gprime = derivative(g)
+    pg = Polynomial(c)
+    pgprime = derivative(pg)
+    # Lower from Polynomial to Function
+    g(x) = pg(x)
+    gprime(x) = pgprime(x)
```

Comment:
@jeremylt It seems we have to do something like this to make the `Tuple{Function, Function}` match because it seems Julia never coerces automatically.

Reply:
That makes sense, I think I've seen this before